src/main/java/com/google/devtools/build/lib/exec/SpawnLogContext.java

```diff
@@ -332,4 +332,8 @@ private Digest computeDigest(
         .setSizeBytes(fileSize)
         .build();
   }
+
+  public boolean needSort() {
```

Comment:
Nit: `shouldSort` is more in line with how we call this kind of method elsewhere in the codebase.